.github/workflows/create-smoke-test-build.yml

```diff
@@ -0,0 +1,18 @@
+name: sync-android-pr
+on:
+  pull_request:
+    types: [labeled]
+jobs:
+  build:
+    if: ${{ github.event.label.name == 'sync-android-pr' }}
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create a WordPress-Android PR to build WordPress-Android with your Current Gutenberg Mobile Changes.
+        env:
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          PR_HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+        run: |
+          npm run sync-pr:android -- "--pr=$PR_NUMBER --sha=$PR_HEAD_SHA"
```

Comment:
Are we still passing in the --sha to the script? 
It seems like it was removed.

Reply:
No, I just haven't finished the changes to the workflow and pushed them up yet.

I was playing around with the workflow toward the end of my day. I just haven't finished testing.

```
      - name: Create a WordPress-Android (WPA) PR containing changes from a Gutenberg Mobile (GBM) PR.
        env:
          PR: ${{ github.event.pull_request.number }}
          GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}
        run: |
          npm run sync:android -- "--pr=$PR"
        if: ${{ github.event.label.name == 'sync:android' }}
      - name: Create a WordPress-Android (WPA) PR containing changes from a Gutenberg Mobile (GBM) Tag.
        env:
          TAG: ${{ github.ref_name }}
          GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}
        run: |
          npm run sync:android -- "--tag=$TAG"
        if: ${{ github.event_name == 'create' && startsWith(github.ref, 'refs/tags/') }}
```